app/assets/javascripts/app/components/comments_list/comments_list.js

```diff
@@ -27,7 +27,7 @@
     ctrl.cancelEdit = cancelEdit;
 
     function updateComment(comment) {
-      var updated = DS.createInstance('discussionComment', comment, { useClass:true });
+      var updated = DS.createInstance('comment', comment, { useClass:true });
```

Comment:
I'd really like these random instance creations from an object that's being pass around in the views not to happen. Couldn't the comment passed in already be a DS instance?
We've already discussed this more thoroughly in person.